Source/driver/Castro_advance_sdc.cpp

```diff
@@ -203,8 +203,15 @@ Castro::do_advance_sdc (Real time,
     // update to
     if (m < SDC_NODES-1) {
       do_sdc_update(m, m+1, dt); //(dt_sdc[m+1] - dt_sdc[m])*dt);
+
+      // we now have a new value of k_new[m+1], do a clean_state on it
+      MultiFab::Copy(S_new, *(k_new[m+1]), 0, 0, S_new.nComp(), 0);
```

Comment:
We can't call clean_state directly on k_new[m+1]?

Reply:
huh... I think you are right.  Maybe I was thinking we needed a ghost fill, but that'll happen inside clean_state.

Reply:
fixed.